src/main/java/singly_linked_list/ListService.java

```diff
@@ -1,33 +1,216 @@
 package singly_linked_list;
 
-public class ListService {
+import java.util.Scanner;
 
+public class ListService implements ServiceOperations{
+    private Scanner scanner;
+
+    public ListService(){
+        scanner = new Scanner(System.in);
+    }
+
+
+    @Override
     public Node createLifo() {
-        return null;
+        int info;
+        Node cur;       //current
+        Node top = null;    //top
+        info = scanner.nextInt();
+        while (info != 0){
+            cur = new Node();
+            cur.info = info;        //cur.setInfo(info)
+            cur.next = top;
+            top = cur;
+            info = scanner.nextInt();
+        }
+        return top;
     }
 
+
+    @Override
     public Node createFifo() {
-        return null;
+        Node top = null, cur;
+        int num = scanner.nextInt();
+        if(num != 0){
+            top = new Node();
+            top.info = num;
+            top.next = null;
+            cur = top;
+            num = scanner.nextInt();
+            while(num != 0){
+                cur.next = new Node();
+                cur = cur.next;
+                cur.info = num;
+                num = scanner.nextInt();
+            }
+            cur.next = null;
+        }
+        return top;
     }
 
-    public void printList(Node top) {
 
+    @Override
+    public void printList(Node top) {
+        if(top == null){
+            System.out.println("The list is empty.");
+            return;
+        }
+        while(top != null){
+            System.out.print(top.info + " ");
+            top = top.next;
+        }
+        System.out.println("\n");
     }
 
-    public void printListRecursively(Node top) {
 
+    @Override
+    public void printListRecursively(Node top) {
+        if(top == null){
+            System.out.print("\n");
+            return;
+        }
+        System.out.print(top.info + " ");
+        printListRecursively(top.next);
     }
 
-    public Node remove(Node top, int toRemove) {
+
+    @Override
+    public Node remove(Node top, int toDelete) {
+        Node cur;
+        Node prev;
+        if(top == null){
+            return null;
```

Comment:
It will be more correct if we will throw exception in this case

Reply:
Corrected.